web/src/frontend/src/nav-soknad/components/validering/Feiloppsummering.tsx

```diff
@@ -1,9 +1,23 @@
 import * as React from "react";
 import { Undertittel } from "nav-frontend-typografi";
 import "./feiloppsummering.css";
-import { Valideringsfeil } from "../../redux/types";
+import { Valideringsfeil } from "../../validering/types";
 
-const scrollToElement = (element: HTMLElement) => {
+const getElementFromFaktumKey = (faktumKey: string): HTMLElement => {
+	if (document.getElementById(faktumKey)) {
+		return document.getElementById(faktumKey);
+	}
+	const namedElements = document.getElementsByName(faktumKey);
+	if (namedElements && namedElements.length > 0) {
+		return namedElements[0];
+	}
```

Comment:
Hva om man har to barn? Enten bruke id, den er unik
Evt. kan hvert element som får valideringsfeil sende inn en handler til riktig dom-node som skal ha fokus direkte

Reply:
Ja, vi må støtte det, men tar det steg for steg nå.